src/config.xml

```diff
@@ -3545,33 +3545,56 @@ to be found in the default search path.
 ]]>
       </docs>
     </option>
-    <option type='string' id='DOT_FONTNAME' format='string' defval='Helvetica' depends='HAVE_DOT'>
+    <option type='string' id='DOT_GRAPH_ATTR' format='string' defval='fontname=Helvetica, fontsize=10' depends='HAVE_DOT'>
       <docs>
-<![CDATA[
- When you want a differently looking font in the dot files that doxygen generates
- you can specify the font name
- using \c DOT_FONTNAME. You need to make sure dot is able to find the font,
- which can be done by putting it in a standard location or by setting the
- \c DOTFONTPATH environment variable or by setting \ref cfg_dot_fontpath "DOT_FONTPATH" to the
- directory containing the font.
-]]>
+        <![CDATA[
+        <a href=https://graphviz.org/doc/info/attrs.html>Node, Edge and Graph Attributes specification</a>
+        ]]>
```

Comment:
A better description of the purpose is required here. The current link is good as background reference but not sufficient (some people will have problems finding it as they are not connected to the Internet).

Reply:
Added

Reply:
I think a little bit more explanation should be present here.